### PR TITLE
feat: add git.toggleVirtualText command

### PR DESCRIPTION
**Pull request by antoinemadec**
I think it's a nice feature to add.
It was already requested in #80.

The code might not be the cleanest as I am not familiar with JS.
I tried to mimic what was already there. The feature works fine for me.

Thanks for the great extension!
Antoine

### CODE REVIEW

File: src/manager.ts [hunk 1]
```diff
@@ -185,6 +197,17 @@ export default class DocumentManager {
     }
   }
 
+  public async toggleVirtualText(): Promise<void> {
+    let enabled = this.virtualText
+    this.config.update('addGBlameToVirtualText', !enabled, true)
```
Comment (chemzqm) [inline src/manager.ts:202]: We don't recommended update user's configuration when possible.

**Comment by antoinemadec** [comment]
@chemzqm , did you have time to look at this?
Is this a feature that you'd consider add to ``coc-git```?

**Review by chemzqm — CHANGES_REQUESTED** [review]

**Comment by chemzqm** [comment]
You should implement show and hide virtualText from coc-git, instead of change user configuration.

**Comment by chemzqm** [comment]
`call coc#config('git.addGBlameToVirtualText', v:false)` could work now.

**Pull request closed without merging**